app/json/slate_configs.json

```diff
@@ -272,6 +272,7 @@
           "recit-personalized/curated"
         ],
         "rankers": [
+          "user-impression-filter",
```

Comment:
Are you planning to monitor the effect on this slate, before applying this ranker more widely?